inc/class-upgrade.php

```diff
@@ -611,6 +611,19 @@ private function upgrade_772() {
 		}
 	}
 
+	/**
+	 * Removes the post meta fields for a given meta key.
+	 *
+	 * @param string $meta_key The meta key.
+	 *
+	 * @return void
+	 */
+	private function delete_post_meta( $meta_key ) {
+		global $wpdb;
+
+		$wpdb->delete( $wpdb->postmeta, array( 'meta_key' => $meta_key ), array( '%s' ) );
```

Comment:
We should at least set the `last_changed` cache key for `postmeta` as it happens in `delete_post_meta_by_key()`. This is inexpensive and at least helps keeping the cache up to date partly.